packages/component-library/src/InitialFocus.ts

```diff
@@ -23,7 +36,12 @@ export function InitialFocus({ children }: InitialFocusProps) {
       setTimeout(() => {
```

Comment:
I don't think we use ScopeTab in the code anymore so maybe this setTimeout can now be removed to simplify the tests

Reply:
This can be done on a separate PR